html/index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+  <head>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <title>Webdev</title>
+    <!-- Google fonts -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Dancing+Script|Montserrat:400,400i,700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Css -->
+    <link rel="stylesheet" href="../styles/styles.css" />
+  </head>
+  <body>
+    <div class="landing-container">
+      <div class="landing-content">
+        <div class="description">
+          <h1>Bunbun's</h1>
+          <p>At the heart of great taste. Since 1905.</p>
+        </div>
+        <div>
+          <a href="./products.html" class="btn">See the goods</a>
+        </div>
+      </div>
+    </div>
+    <footer>
+      <div class="footer-left">
```

Comment:
it usually is better to name the classes by what they do in the app not by where there are placed (left, right) because the design might change or the view might be changed on mobile
you can put something like '.contact-list', '.contact-links', '.footer-description' or '.footer .description'